src/stylesheets/common/_buttons.scss

```diff
@@ -22,12 +22,17 @@ input[type="search"] + .btn {
 
 // Dark purple button
 .btn-mapzen {
-  background-color: nth($mz-darkpurples, 1);
-  color: #ffffff;
+  @extend .btn-primary;
+}
+
+.btn-primary {
+  background-color: $mz-darkpurple-1;
```

Comment:
Ah, I was just looking at the staging branch and missed those. (good reminder to water that branch)

Thanks again. This looks great!